api/src/main/java/io/druid/data/input/impl/CSVParseSpec.java

```diff
@@ -80,23 +103,23 @@ public void verify(List<String> usedCols)
   @Override
   public Parser<String, Object> makeParser()
   {
-    return new CSVParser(Optional.fromNullable(listDelimiter), columns);
+    return new CSVParser(Optional.fromNullable(listDelimiter), columns, hasHeaderRow, skipHeaderRows);
```

Comment:
(_Optional_ comment) Note that `Optional` is not recommended to be used as a method/constructor parameter: http://stackoverflow.com/a/26328555/648955.